### PR TITLE
PRChecker: limit num of last commits that gets logged.

### PR DESCRIPTION
**Pull request by cPhost**
what the title say really...

### CODE REVIEW

File: test/unit/pr_checker.test.js [hunk 1]
```diff
@@ -307,6 +308,42 @@ describe('PRChecker', () => {
...
     });
+
+    it('should only log last three commits if multiple', () => {
+      const logger = new TestLogger();
+      const { commits, comment } = mulipleCommitsAfterCi;
+
+      const expectedLogs = {
+        warn: [
+          [ 'Commits pushed after the last Full CI run:' ],
+          [ 'last 3 commits...' ],
```
Comment (joyeecheung) [inline test/unit/pr_checker.test.js]: Can you merge this line with the line above? `Commits pushed after the last Full CI run. The last 3 commits are..`

Comment (cPhost) [reply test/unit/pr_checker.test.js]: @joyeecheung i will fix it.

**Comment by joyeecheung** [comment]
@Tiriel This is for the CI run detection, not the reviews

**Comment by Tiriel** [comment]
My bad, thanks for the clarification!

**Comment by cPhost** [comment]
@Tiriel the merge conflict was because new `cli` functionality. it does not matter if other PR get merged or not this PR will have no effect. Git only cares about lines not the whole files.  Does that clarify it for you?

**Comment by Tiriel** [comment]
@cPhost Actually that's precisely what I was talking about. But that was because de diff was showing me that you had changed the same line in the two PRs. Maybe I didn't see properly or I openned the same PR twice, or there was a bug. Dunno.

Sorry about that :smile:

File: test/unit/pr_checker.test.js (hunk 1 above)
Comment (cPhost) [reply test/unit/pr_checker.test.js]: done

**Comment by cPhost** [comment]
Merging it now since i wanted to update that PR for `checkCI`.

**Pull request merged**